maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Models/SwipeRequest.swift

```diff
@@ -30,5 +32,6 @@ struct SwipeRequest: Decodable {
             y: try container.decode(Double.self, forKey: .endY)
         )
         duration = try container.decode(Double.self, forKey: .duration)
+        appIds = try container.decodeIfPresent([String].self, forKey: .appIds)
```

Comment:
For safety and reduction of optionals in models, it may be better to create a `swipeV3()` and `SwipeRequestV3`.
WDYT?

Reply:
Hmmm, open to do that but do we know when we should completely delete swipe v1 and v2 🤔

Reply:
When both cli and cloud are using V3, the others can be removed

Reply:
Umm, I was looking at the scope and it's indeed just a couple of line changes do you still think v3 makes sense for that?